CHANGELOG.md

```diff
@@ -2,11 +2,29 @@
 All notable changes to [Solang](https://github.com/hyperledger/solang/)
 will be documented here.
 
-## Unreleased
+## v0.3.2 Brasília
+
+The language server is much improved, and many fixes all over.
+
+### Added
+- Go to definition is now implemented in the language server. [chioni16](https://github.com/chioni16)
+- The parser has been updated to be compatible with Ethereum Solidity v0.8.21. [seanyoung](https://github.com/seanyoung)
 
 ### Fixed
```

Comment:
I fixed the overflowing problem in the `tranfer` function and avoided overwriting the signer and writer flags. Are these bug fixes worth mentioning?

Reply:
The changelog should really list things which are user-affecting, in a consumable way. This is a security change, so yes this can be mentioned.

Reply:
I've added an entry for these two items